MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: Getting Started with .NET MAUI DataForm control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI DataForm (SfDataForm) control and its basic features.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Getting Started with .NET MAUI DataForm (SfDataForm)
+
+This section provides a quick overview of how to get started with the .NET MAUI DataForm(SfDataForm) for .NET MAUI and also provides a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
+
+## Creating an application using the .NET MAUI DataForm
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfDataForm to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.DataForm] and then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.DataForm` in XAML or C# code.
+
+4. Initialize `SfDataForm.`
+
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3 5" %}
+
+    <ContentPage   
+            
+            xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm" >
+
+            <dataForm:SfDataForm />
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="1 9 10" %}
+
+    using Syncfusion.Maui.DataForm;
+    . . .
+
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+    {
+        InitializeComponent();
+        SfDataForm dataForm = new SfDataForm();
+        this.Content = dataForm;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
```

Comment:
The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.

Reply:
nuget cases changed